pkg/workflow/argo.go

```diff
@@ -329,7 +329,7 @@ func (a *argo) generateReverseWorkflow(ctx context.Context, l logr.Logger, nodes
 
 	updateWorkflowTemplates(a.rwf, entry)
 
-	updateWorkflowTemplates(a.rwf, defaultReverseExecutor())
+	updateWorkflowTemplates(a.rwf, defaultExecutor(helmReleaseReverseExecutor, "delete"))
```

Comment:
Let's define the action strings ("install" & "delete") as `type ExecutorAction string`  and create `const`s for each one.